nupic/research/frameworks/vernon/handlers.py

```diff
@@ -404,8 +404,7 @@ def create_lr_scheduler(cls, config, optimizer, total_batches):
                 lr_scheduler_args=lr_scheduler_args,
                 steps_per_epoch=total_batches)
 
-    @classmethod
-    def create_loaders(cls, config):
+    def create_loaders(self, config):
```

Comment:
I think I successfully reverted the change. `create_loaders` typically returns 2 torch dataloaders, but since meta-continual learning is a bit more complex, the `create_loaders` in the MetaContinualLearningExperiment class was returning 4 torch dataloaders. I've changed so that it will only return 2, and added a new class method `create_fast_slow_loaders`. Will push changes shortly.